src/app/containers/time-limited-content-info/time-limited-content-info.component.ts

```diff
@@ -0,0 +1,63 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import { Store, createSelector } from '@ngrx/store';
+import { filter, interval, map, of, switchMap } from 'rxjs';
+import { fromItemContent } from 'src/app/items/store';
+import { DurationAsCountdownPipe } from 'src/app/pipes/duration';
+import { isInfinite } from 'src/app/utils/date';
+import { Duration } from 'src/app/utils/duration';
+import { isNotUndefined } from 'src/app/utils/null-undefined-predicates';
+
+/**
+ * The current result. `null` if we know for sure there is no current result. `undefined` if it is not known yet.
```

Comment:
Did you mean `:` instead `.` ?

Reply:
no, otherwise we may thing a current can only be null or undefined. do you prefer ";" or ","?

Reply:
ah, got it, probably `,`